src/components/webpanel/webpanel.ts

```diff
@@ -1,19 +1,18 @@
 import * as vscode from 'vscode';
+import { LogsPanel } from '../logs/logsWebview';
```

Comment:
`WebPanel` is also used by the Describe webview; it shouldn't depend on `LogsPanel`.